core/sqf/src/stfs/stfs_stub.cpp

```diff
@@ -115,6 +115,7 @@ namespace STFS_stub {
   } stfs_directories;
 
   stfs_directories stfsDirsHDD_, stfsDirsSSD_;
+  char scratchCQDLocationString_[MAX_STFS_LOCATIONS*STFS_PATH_MAX - STFS_SUFFIX+1]="";
```

Comment:
This seems to allow space for just under 32*4096 = 131072 bytes, but I notice the DEFAULTS table limits CQD values to 1000 bytes. Just wondering what the maximum CQD value length at the command line is and whether you'll ever even get close to 131072 here.